rackcity/models/asset_change_planner.py

```diff
@@ -0,0 +1,79 @@
+from django.db import models
+from .asset import validate_hostname, validate_owner
+from django.core.validators import MaxValueValidator, MinValueValidator
+from django.core.exceptions import ValidationError, ObjectDoesNotExist
+from .it_model import ITModel
+from .rack import Rack
+from .asset import Asset
+from .change_plan import ChangePlan
+
+
+class AssetCP(models.Model):
+    related_asset = models.ForeignKey(
+        Asset,
+        on_delete=models.SET_NULL,
+        null=True,
+        blank=True,
+    )
+    ## TODO: related_decomissioned_asset
```

Comment:
What will this field be?

Reply:
this will be the same idea as related_asset but will point to a decomissionedAsset object once it exists